plugins/modules/mount.py

```diff
@@ -825,7 +898,11 @@ def main():
                 module.fail_json(
                     msg="Error making dir %s: %s" % (name, to_native(e)))
 
-        name, backup_lines, changed = _set_mount_save_old(module, args)
+        # ephemeral: completely ignore fstab
+        if state != 'ephemeral':
+            name, backup_lines, changed = _set_mount_save_old(module, args)
+        else:
+            name, backup_lines, changed = args['name'], [], False
```

Comment:
The function `_set_mount_save_old` records the mount in the *fstab* file. This step must be bypassed for `ephemeral` state.